ipaserver/plugins/idviews.py

```diff
@@ -94,7 +95,7 @@ class idview(LDAPObject):
     container_dn = api.env.container_views
     object_name = _('ID View')
     object_name_plural = _('ID Views')
-    object_class = ['ipaIDView', 'top']
+    object_class = ['ipaIDView', 'ipaNameResolutionData', 'top']
```

Comment:
Correct, it is auxilliary object class, so it should be in the `possible_object_class` and `default_attributes` should contain `ipaNameResolutionOrder` to be able to return the value if it is there.